Core/clim-core/dialog-views.lisp

```diff
@@ -251,6 +251,36 @@
                                   :value-changed-callback #'value-changed-callback))))
 
 
+;;;; text-editor-view
+
+(define-presentation-method accept-present-default
+    ((type string) stream (view text-editor-view) default default-supplied-p
+     present-p query-identifier)
+  (unless default-supplied-p
+    (setf default ""))
+  (let* ((slots (c2mop:class-slots (find-class 'text-editor-view)))
```

Comment:
I think this should be done using a generic function as described in an earlier annotation.

In case that is not possible: 
* `(class-slots (find-class 'text-editor-view))' should not be called multiple times. 
* Doesn't `class-slots` require the class to be finalized?

Reply:
I've changed it.